mono/utils/mono-threads.c

```diff
@@ -121,7 +121,7 @@ void
 mono_threads_add_to_pending_operation_set (MonoThreadInfo* info)
 {
 	THREADS_SUSPEND_DEBUG ("added %p to pending suspend\n", mono_thread_info_get_tid (info));
-	++pending_suspends;
+	InterlockedIncrement (&pending_suspends);
```

Comment:
This should not be needed as there is at most one thread accessing this variable.